frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -517,4 +437,77 @@ const LabworkStep = ({ protocol, step, stepSamples }: LabworkStepPageProps) => {
 	)
 }
 
-export default LabworkStep
+
+export interface SelectionTabProps {
+	stepSamples: LabworkStepSamples
+	step: Step
+	protocol: Protocol | undefined
+	selection: WorkflowSamplesTableProps['selection']
+	setSortBy: WorkflowSamplesTableProps['setSortBy']
+}
+
+function SelectionTab({stepSamples, step, protocol, selection, setSortBy}: SelectionTabProps) {
+	const dispatch = useAppDispatch()
+
+	const [pageSize, setPageSize] = useState(10)
```

Comment:
what's up with this pesky hardcoded page size of 10 ?

Reply:
Oh nooooo